translations.txt

```diff
@@ -595,63 +595,63 @@ Your Trial key includes all the functionality and device support of a Pro key=De
-Upgrade keys to higher editions=
-While we strive to respond to all requests as quickly as possible please allow for up to 3 business days for a response=
-You have an open pop-up=
-You may purchase a license key corresponding to this USB Flash device to continue using this installation=
-Your flash drive is corrupted or offline=
+Please copy the correct key file to the bootconfig directory on your USB Flash boot device or choose Purchase Key or Replace Key=Bitte kopieren Sie die korrekte Lizenzschlüssel Datei in den /boot/config Ordner auf dem USB Flash Boot-Datenträger oder wählen Sie Schlüssel kaufen oder ersetzen
+Please copy the correct key file to the bootconfig directory on your USB Flash boot device or choose Purchase Key=Bitte kopieren Sie die korrekte Lizenzschlüssel Datei in den /boot/config Ordner auf dem USB Flash Boot-Datenträger oder wählen Sie Schlüssel kaufen
+Please try a different USB Flash device=Bitte wählen sie ein anderes USB Flash-Laufwerk
+Post your diagnostics in the forum for help=Bitte posten Sie die Diagnose im Forum damit wir Ihnen helfen können
+See also here=Siehe auch
+The key file should be located in the bootconfig directory on your USB Flash boot device=Die Lizenzschlüssel-Datei muss in den /boot/config Ordner auf Ihren USB Boot-Laufwerk
+This can occur as a result of transferring your license key to a replacement USB Flash device, and you are currently booted from your old USB Flash device=Dies kann auftreten wenn sie Ihre Lizenz transferiert haben zu einem neuen USB Flash-Laufwerk und von dem alten Flash-Laufwerk booten
+Upgrade keys to higher editions=Upgraden Sie ihre Lizenz zu einer höheren Edition
+While we strive to respond to all requests as quickly as possible please allow for up to 3 business days for a response=Während wir versuchen alle Anfragen so schnell wie möglich zu beantworten, geben Sie uns bitte bis zu 3 Arbeitstage
+You have an open pop-up=Sie haben ein offenes Pop-Up
+You may purchase a license key corresponding to this USB Flash device to continue using this installation=Sie müssen eine passende Lizenz für dieses Flash-Laufwerk kaufen damit sie diese Installation weiter nutzen können
+Your flash drive is corrupted or offline=Ihr Flash-Laufwerk ist beschädigt oder entfernt
 
 ; New entries (11) - 2021-05-09
-Blacklisted USB Flash GUID=
+Blacklisted USB Flash GUID=Gesperrte USB-Flash-GUID
 complete=
-Copying=
+Copying=Kopieren
 object=
-Password=
-Purchase Key=
-Subject=
-Thank you for choosing Unraid OS=
-There is a physical problem accessing your USB Flash boot device=
-Trial=
-Upgrade Key=
+Password=Passwort
+Purchase Key=Schlüssel kaufen
+Subject=Betreff
+Thank you for choosing Unraid OS=Danke das Sie sich für Unraid OS entschieden haben
+There is a physical problem accessing your USB Flash boot device=Es gibt Probleme auf den Flash Boot-Datenträger zuzugreifen
+Trial=Probe
+Upgrade Key=Lizenz erweitern
```

Comment:
In der ursprünglichen Übersetzung wurde "Lizenz Upgraden" verwendet also würde hier: "Lizenz Upgraden" passen.